### PR TITLE
Jakarta Persistence 2.2

### PR DESCRIPTION
**Pull request by lukasj**
Signed-off-by: Lukas Jungmann <lukas.jungmann@oracle.com>

PR#1:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [x] Specification PDF in the form of wombat_x.y.pdf
- [x] Specification HTML in the form of wombat_x.y.html
- [x] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md
- [x] For a Release Review, a summary that a Compatible Implementation
      is complete, passes the TCK, and that the TCK includes sufficient
      coverage of the specification. The TCK users guide MUST include
      the instructions to run the compatible implementations used to
      validate the release.  Instructions MAY be by reference.
- [x] The URL of the OSSRH staging repository for the api, javadoc:
      https://oss.sonatype.org/content/repositories/staging/jakarta/persistence/jakarta.persistence-api/2.2.3/
- [x] The URL of the staging directory on downloads.eclipse.org for the proposed EFTL TCK binary:
      http://download.eclipse.org/ee4j/jakartaee-tck/jakartaee8-eftl/promoted/eclipse-persistence-tck-2.2.0.zip
- [x] The URL of the compatibility certification request issue:
      https://github.com/eclipse-ee4j/jpa-api/issues/228

Include the following in PR#2:
- [x] Specification JavaDoc in the wombat/x.y/apidocs directory.

### CODE REVIEW

**Comment by dblevins** [comment]
Spec PDF looks good.

File: persistence/2.2/_index.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: "Jakarta Persistence Specification 2.2"
+date: 2019-07-19T21:45:02+0200
+summary: "First release for Jakarta EE 8"
+---
+The Jakarta Persistence API is the Java(R) API for the management of persistence
+and object/relational mapping in Jakarta EE and Java SE environments.
```
[the remark this answers is not in the document]
Comment (lukasj) [reply persistence/2.2/_index.md]: fixed

File: persistence/2.2/_index.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+|Red Hat                |         |         |          |
+|Tomitribe              |         |         |          |
+|EE4J PMC               |         |         |          |
+|Participant Members    |         |         |          |
+|Committer Members      |         |         |          |
+
+
+# Compatible Implementations
+
+* EclipseLink 2.7.4
```
[the remark this answers is not in the document]
Comment (lukasj) [reply persistence/2.2/_index.md]: fixed

File: persistence/2.2/_index.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
+title: "Jakarta Persistence Specification 2.2"
+date: 2019-07-19T21:45:02+0200
```
[the remark this answers is not in the document]
Comment (lukasj) [reply persistence/2.2/_index.md]: removed

**Comment by starksm64** [comment]
Needs SHA updated

**Comment by dblevins** [comment]
@lukasj The staged API binaries will be expired and deleted automatically by Nexus OSSRH on Sunday, Sept 1 st.  Can you publish this api jar to Maven Central at your earliest convenience?

**Comment by dblevins** [comment]
Note, this repo should be dropped and not published as we are not putting the EFSL specifications in Maven Central.

 - https://oss.sonatype.org/content/repositories/jakartapersistence-1017/

**Comment by starksm64** [comment]
- [x] The designated specification committee representative adds this final checklist to the main PR.
 - [ ] The specification committee representative merges the specification PRs.
 - [ ] The designated specification committee representative calculates the staged EFTL TCK signature and promotes it to the committee download area
  using the https://ci.eclipse.org/jakartaee-spec-committee/job/promote-release/ job.
 - [ ] The specification project member who created the staging release promotes the specification api jars to maven central. An example release job script can be found here https://wiki.eclipse.org/MavenReleaseScript.
 - [ ] The EMO updates the specification page with the ballot results.
This list goes on the committed spec index page.
 - [ ] The specification project team should go through the merged spec website page to verify all the links are valid.
 - [ ] The specification project team should approve the compatibility request and send an email to tck@eclipse.org for approval of the compatible implementation for trademark usage.
 - [ ] The specification project team should merge any final release branch as approriate for the branch management for the project.

**Comment by bshannon** [comment]
@lukasj Please confirm that the last two project team items in the checklist have been done.

**Pull request merged**